components/favoris/offres/OffreRow.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { Tag } from 'components/ui/Tag'
+import { Offre } from 'interfaces/favoris'
+
+export default function OffreRow({ offre }: { offre: Offre }) {
+  return (
+    <div
+      role='row'
+      className='table-row h-20 text-sm rounded-[6px] shadow-s hover:bg-primary_lighten group'
```

Comment:
pas de besoin de fixer la hauteur
```suggestion
      className='table-row text-sm rounded-[6px] shadow-s hover:bg-primary_lighten group'
```

Reply:
En fait, on veut que quelque soit la longueur du titre, on ait une cellule d'au moins 80 px. C'est la contrainte de la maquette. Tu vois une autre manière de faire ?

Reply:
Dans ce cas, soit :
- c'est un `min-height` plutôt qu'une `height`
- on peut truncate/ellispe le titre pour qu'il soit tout le temps sur une seule ligne